Example/iamport-ios/ViewModel.swift

```diff
@@ -46,7 +46,7 @@ public class ViewModel: ObservableObject, Then {
     @Published var isCert: Bool = false
     @Published var showResult: Bool = false
     @Published var CardDirectCode: String = ""
-    var iamPortResponse: IamPortResponse?
+    var iamPortResponse: IamportResponse?
```

Comment:
```suggestion
    var iamportResponse: IamportResponse?
```
아직 고쳐지지 않은 것들이 조금 있군요